modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ScheduleContainer.scss

```diff
@@ -1,10 +1,17 @@
 @import 'atlas-variables';
 
 .lfr-object__entries-schedule-panel {
-	.disabled {
-		.text-warning {
-			color: $secondary-l1;
-			opacity: 1;
+	&-content {
+		.disabled {
+			.text-warning {
+				color: $secondary-l1 !important;
```

Comment:
Hi @larissacribeiro, why this important is needed?

Reply:
as we discussed in person, i've changed this to modify the color directly on FieldBase. please, take another look when you get the chance!

Reply:
Thanks for the change @larissacribeiro.

Waiting for the front-end review below [here](https://github.com/liferay-frontend/liferay-portal/pull/4970)